CHANGELOG.md

```diff
@@ -3,8 +3,13 @@
 [PyPI History][1]
 
 [1]: https://pypi.org/project/demisto-py/#history
+
+
+## 3.0.3
+* Support `upload_content_packs` to upload zipped pack to marketplace.
```

Comment:
I think you can unite this with 3.0.2 as it looks like 3.0.2 wasn't released yet.